clippy_lints/src/matches.rs

```diff
@@ -1509,6 +1509,7 @@ mod redundant_pattern_match {
     use rustc_errors::Applicability;
     use rustc_hir::{Arm, Expr, ExprKind, MatchSource, PatKind, QPath};
     use rustc_lint::LateContext;
+    use rustc_span::sym;
```

Comment:
```suggestion
```

Shouldn't be necessary, since it is already imported above.

Reply:
It's needed for the inner mod.